pegasus/sites.v3/code.org/public/maker/circuitplayground.haml

```diff
@@ -0,0 +1,97 @@
+---
+title: The Circuit Playground | Code.org
+theme: responsive_full_width
+---
+
+%link{href: '../shared/css/design-system-pegasus.css', rel: 'stylesheet'}
+%link{href: '/css/generated/maker-styles.css', rel: 'stylesheet'}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
+
+-# FontAwesome icons
+- icon_stars = "fa-solid fa-stars"
+- icon_puzzle_piece = "fa-solid fa-puzzle-piece"
+- icon_circle_check = "fa-solid fa-circle-check"
+- icon_circle_xmark = "fa-solid fa-circle-xmark"
+- icon_flag = "fa-solid fa-flag-checkered"
+
+%section.skinny-banner.no-padding-bottom
+  .wrapper
+    = view :maker_skinny_banner
+
+%section.intro
+  .wrapper
+    %h1 The Circuit Playground
+    %p.h5-lookalike-style The Circuit Playground is family of small microcontroller boards with LEDs, buttons, and sensors built in.
+    %div.flex-container
+      %ul.icon-bullet-list.col-60
+        %li
+          %div
+            %i{class: "#{icon_stars}"}
+          .text-wrapper
+            %h3 Designed specifically for novice programmers
+            %p Enables students to get up and running quickly with physical computing without worrying about many of the traditional barriers to entry.
+        %li.no-margin-bottom
+          %div
+            %i{class: "#{icon_puzzle_piece}"}
+          .text-wrapper
+            %h3 Works with App Lab
+            %p.no-margin-bottom Using App Lab students can quickly build apps that communicate with external hardware using our block-to-text app development environment.
+      %figure.col-25{style: "margin-top: 2em; text-align: center"}
+        %img{src: "/images/maker/circuitplayground-no-bg.gif", alt: ""}
+    .clear
+
+%section.bg-neutral-light.device-setup
+  .wrapper
+    %h2 Device setup requirements
+    %p The Circuit Playground is a series of circuit boards produced by Adafruit with a variety of sensors and outputs built in.
+    %p <strong>There are three variants of the Circuit Playground, two are compatible with App Lab and one of them is not:</strong>
+    %ul.icon-bullet-list{style: "margin-top: 0"}
+      %li
+        %div
+          %i{class: "#{icon_circle_check}", style: "color: #3EA33E"}
```

Comment:
This is color-coding weird in the Github browser, making me think there's a syntax error, but I don't see any errors here ... commenting and will come back to this